src/main/java/de/tum/cit/aet/domain/Simulation.java

```diff
@@ -45,10 +45,24 @@ public class Simulation {
     @Column(name = "user_range")
     private String userRange;
 
+    @Deprecated
```

Comment:
just curious, why is this deprecated?

Reply:
IDEType basically is an enum with types "OFFLINE" and "ONLINE_IDE" the percentage settings below obsolete this setting. I left it for now for to not introduce any breaking database changes when using a different version of the benchmarking tool. Should I directly remove it here too?

Reply:
Ah get it thanks! It sounds good to me leave with @Deprecated for now maybe